pkg/graph/schema.resolvers.go

```diff
@@ -1752,8 +1752,22 @@ func (r *queryResolver) Roles(ctx context.Context, cedarSystemID string, roleTyp
 	return cedarRoles, nil
 }
 
+func (r *queryResolver) Urls(ctx context.Context, cedarSystemID string) ([]*models.CedarURL, error) {
+	cedarURLs, err := r.cedarCoreClient.GetURLsForSystem(ctx, cedarSystemID)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(cedarURLs) == 0 {
+		return nil, &apperrors.ResourceNotFoundError{Err: fmt.Errorf("no URLs found"), Resource: []*models.CedarURL{}}
```

Comment:
How come you return an error here and not an empty slice?

Reply:
I was following the convention of some of the other `queryResolver` methods; `Deployments()` and `Roles()` both do the same thing. That said, this behavior isn't consistent across all our resolvers for top-level queries; this returns an error (which can be tested with `cedarSystemId:"408-4151-0"`), but the `cedarThreat` query just returns an empty array with the following query:
```
query {
  cedarThreat(cedarSystemId:"408-995-0") {
    id
  }
}
```

We should probably pick a convention and stick to it; not sure which way to go, though, or if it matters for this PR.